tools/topology/topology2/cavs/CMakeLists.txt

```diff
@@ -26,17 +26,17 @@ PREPROCESS_PLUGINS=nhlt,NHLT_BIN=nhlt.bin,USE_CHAIN_DMA=true"
 HDA_CONFIG=mix,NUM_DMICS=4,PDM1_MIC_A_ENABLE=1,PDM1_MIC_B_ENABLE=1,PREPROCESS_PLUGINS=nhlt,NHLT_BIN=nhlt-mtl-hda-mix.bin"
 
 # CAVS SDW topology with passthrough pipelines
-"cavs-sdw\;cavs-sdw"
+"cavs-sdw\;cavs-sdw\;DEEPBUFFER_FW_DMA_MS=100"
```

Comment:
@RanderWang @plbossart this somehow gives you the feeling that we are infact scheduling this pipeline every 100ms. When in fact all we're doing is 1ms LL tasks still right?

Reply:
that's quite possible indeed, and that would be problematic if processing is added.

but for the time being, what we need to verify is that the DMA bursts are larger, e.g. we don't make DMA requests every ms. That would clearly be a problem for D0i3 usages. The path to memory has to be used only when needed.